manifests/i/Inkscape/Inkscape/1.3/Inkscape.Inkscape.installer.yaml

```
@@ -0,0 +1,34 @@
+# Created with Komac v1.9.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Inkscape.Inkscape
+PackageVersion: '1.3'
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+FileExtensions:
+- ai
+- dxf
+- eps
+- odg
+- pdf
+- png
+- ps
+- svg
+- wmf
+ProductCode: '{2C69A8D5-2E44-4F99-BD5E-08536B52F1DA}'
+ReleaseDate: 2023-07-22
+Installers:
+- Architecture: x86
+  InstallerUrl: https://media.inkscape.org/dl/resources/file/inkscape-1.3_2023-07-21_0e150ed6c4-x64_31XBEKV.msi

```

```suggestion
  InstallerUrl: https://media.inkscape.org/dl/resources/file/inkscape-1.3_2023-07-21_0e150ed6c4-x86_K2QrAzI.msi
```

(and of course, the product code and SHA256)